### ISSUE TITLE
[Bug]: all internal tables/database can be delete/drop...etc 

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#5165]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior
delete from mo_user;
![image](https://user-images.githubusercontent.com/113406637/191491918-5b5aa4d9-0d0c-4686-b33e-132e11475da4.png)
drop internal databases 
drop database information_schema;
<img width="427" alt="企业微信截图_158df383-8b8e-4087-ae9b-03c58074d78d" src="https://user-images.githubusercontent.com/113406637/192086552-033221b4-7bdf-4a59-8baa-596e27cd91d5.png">

**Comment by daviszhen** [comment]
if was fixed at #5587

**Comment by heni02** [comment]
confirm, closed
